apps/vs-code-designer/src/app/utils/binaries.ts

```diff
@@ -409,7 +413,7 @@ export async function promptInstallBinariesOption(context: IActionContext) {
   if (vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length > 0) {
     const workspace = await getWorkspaceFolder(context);
     const projectPath = await tryGetFunctionProjectRoot(context, workspace);
-    const message = localize('useBinaries', 'Always validate and install the latest dependency binaries at launch');
+    const message = localize('useBinaries', 'Always validate and install the latest dependency binaries at launch TEST');
```

Comment:
the TEST part should be removed, right?

Reply:
Yep, just made the change so Esther can review the strings 😅